lib/ansible/modules/cloud/google/drive_append_to_sheet.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Chris Blum <chris.n.blum@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1', 'status': ['preview'], 'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: drive_append_to_sheet
+
+short_description: Archive variables in a Google Sheet
+
+version_added: "2.8"
+
+description:
+    - "Using this module it is possible to archive variables in a playbook run into a Google Sheet.
+      This is especially interesting if you are trying to gather facts across lots of servers and
+      you will want to do some statistical analysis on that data later on.
+      A Google account and the Python client library for Google APIs are requirements.
+      Also you must have the Google Sheets API activated and valid credentials for this handy.
+      The necessary API steps can be done here:
+      https://developers.google.com/sheets/api/quickstart/python#step_1_turn_on_the"
+
+options:
+    columns:
+        description:
+            - This is the variable that should be appended to the GSheet
+        required: true
+        aliases: [ 'name' ]
+    sheetID:
+        description:
+            - This is the Sheet ID
+        required: true
+    client_id:
+        description:
+            - This is the Google OAuth client_id
+              It ends with .apps.googleusercontent.com
```

Comment:
Make this two items in your description.
```suggestion
            - The value should end with C(.apps.googleusercontent.com).
```

Reply:
resolved by fb49811